1/bump_version.py

```diff
@@ -188,6 +189,36 @@ def update_version_in_file(version_file_path: str, new_version: str):
             version_file.write(new_content)
 
 
+def add_ticket(changelog_path, new_version):
+    ticket_base_url = os.getenv('PLUGIN_TICKET_URL')
+    if not ticket_base_url:
+        print(f"Not adding the ticket to changelog, because no url (PLUGIN_TICKET_URL) to ticketing was provided.")
```

Comment:
I think the log is not required here, if the user wants it, this will be set. However it should return instead.
```suggestion
        return
```

Reply:
argh forgot the return indeed.